src/Core/Managed/Shared/ActivityExtensions.cs

```diff
@@ -50,6 +38,31 @@ internal static void SetOperationName(this Activity activity, string operationNa
             Debug.Assert(activity != null, "Activity must not be null");
             activity.AddTag(OperationNameTag, operationName);
         }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        private static bool Initialize()
+        {
+            try
+            {
+                Assembly.Load(new AssemblyName("System.Diagnostics.DiagnosticSource, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"));
```

Comment:
can you please have a unit test that will check the version in the string matches the version of actual dependency. So we will keep it in sync and have a verification that will fail if we forgot to update this string